模型前向傳播錯誤: Given groups=1, weight of size [96, 3, 4, 4], expected input[1, 5, 1, 80000] to have 3 channels, but got 5 channels instead
Traceback (most recent call last):
  File "/home/sbplab/JNRLEE/MicDysphagiaFramework/tests/test_swin_transformer_inputs.py", line 223, in analyze_model_input_output
    outputs = model(batch['spectrogram'].to(device))
  File "/home/sbplab/miniconda3/envs/ENT/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1736, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/home/sbplab/miniconda3/envs/ENT/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1747, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/sbplab/JNRLEE/MicDysphagiaFramework/models/swin_transformer.py", line 265, in forward
    features = self.backbone(x)
  File "/home/sbplab/miniconda3/envs/ENT/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1736, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/home/sbplab/miniconda3/envs/ENT/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1747, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/sbplab/miniconda3/envs/ENT/lib/python3.9/site-packages/torchvision/models/swin_transformer.py", line 608, in forward
    x = self.features(x)
  File "/home/sbplab/miniconda3/envs/ENT/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1736, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/home/sbplab/miniconda3/envs/ENT/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1747, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/sbplab/miniconda3/envs/ENT/lib/python3.9/site-packages/torch/nn/modules/container.py", line 250, in forward
    input = module(input)
  File "/home/sbplab/miniconda3/envs/ENT/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1736, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/home/sbplab/miniconda3/envs/ENT/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1747, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/sbplab/miniconda3/envs/ENT/lib/python3.9/site-packages/torch/nn/modules/container.py", line 250, in forward
    input = module(input)
  File "/home/sbplab/miniconda3/envs/ENT/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1736, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/home/sbplab/miniconda3/envs/ENT/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1747, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/sbplab/miniconda3/envs/ENT/lib/python3.9/site-packages/torch/nn/modules/conv.py", line 554, in forward
    return self._conv_forward(input, self.weight, self.bias)
  File "/home/sbplab/miniconda3/envs/ENT/lib/python3.9/site-packages/torch/nn/modules/conv.py", line 549, in _conv_forward
    return F.conv2d(
RuntimeError: Given groups=1, weight of size [96, 3, 4, 4], expected input[1, 5, 1, 80000] to have 3 channels, but got 5 channels instead
